src/app/components/organisms/Tabs/Tabs.sc.ts

```diff
@@ -33,6 +33,7 @@ export const TabHeader = styled.button<TabHeaderProps>`
     border-bottom: 2px solid ${({ theme }): string => theme.colorDisabled};
     cursor: pointer;
     padding: ${({ theme }): string => theme.spacing(1.25, 3)};
+    height: 54px;
```

Comment:
Please use the `theme.spacing` function.
It's being used everywhere (afaik), so better stick to it.